packages/malloy/src/lang/test/test-translator.ts

```diff
@@ -32,109 +33,112 @@ export function pretty(thing: any): string {
   return inspect(thing, { breakLength: 72, depth: Infinity });
 }
 
-const mockSchema: Record<string, StructDef> = {
-  aTable: {
-    type: "struct",
-    name: "aTable",
-    dialect: "standardsql",
-    structSource: { type: "table" },
-    structRelationship: { type: "basetable", connectionName: "test" },
-    fields: [
-      { type: "string", name: "astr" },
-      { type: "number", name: "af", numberType: "float" },
-      { type: "number", name: "ai", numberType: "integer" },
-      { type: "date", name: "ad" },
-      { type: "boolean", name: "abool" },
-      { type: "timestamp", name: "ats" },
-    ],
-  },
-  "malloytest.carriers": {
-    type: "struct",
-    name: "malloytest.carriers",
-    dialect: "standardsql",
-    structSource: {
-      type: "table",
-      tablePath: "malloytest.carriers",
+export function makeMockSchema(
+  dialect = "standardsql"
+): Record<string, StructDef> {
+  return {
+    aTable: {
+      type: "struct",
+      name: "aTable",
+      dialect,
+      structSource: { type: "table" },
+      structRelationship: { type: "basetable", connectionName: "test" },
+      fields: [
+        { type: "string", name: "astr" },
+        { type: "number", name: "af", numberType: "float" },
+        { type: "number", name: "ai", numberType: "integer" },
+        { type: "date", name: "ad" },
+        { type: "boolean", name: "abool" },
+        { type: "timestamp", name: "ats" },
+      ],
     },
-    structRelationship: { type: "basetable", connectionName: "bigquery" },
```

Comment:
Bad diff, but this is just a re-indentation. I moved the mock schema into this constructor (`makeMockSchema`) to enable parameterizing it on `dialect`.